### PR TITLE
Simplify ifdef directives

### PR DESCRIPTION
**Pull request by maximiliankolb**
Follow-up PR to https://github.com/theforeman/foreman-documentation/pull/1960 @bangelic

### CODE REVIEW

File: guides/common/assembly_administering-hosts.adoc [hunk 1]
```diff
@@ -24,13 +24,9 @@ include::modules/proc_changing-the-host-group-of-a-host.adoc[leveloffset=+1]
 
 ifdef::katello,satellite,orcharhino[]
 include::modules/proc_adding-a-host-to-a-host-collection.adoc[leveloffset=+1]
-endif::[]
 
-ifdef::katello,satellite[]
 include::modules/proc_using-the-host-details-tab.adoc[leveloffset=+1]
```
Comment (adamlazik1) [inline guides/common/assembly_administering-hosts.adoc:28]: Is this one also for orcharhino now?

**Review by adamlazik1 — APPROVED** [review]
LGTM

**Comment by adamlazik1** [comment]
Any cherry-picks?

**Comment by maximiliankolb** [comment]
> Any cherry-picks?

Yes, to 3.5 please (same as original PR).

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   cf4655b7e..9c52482ce  3.5 -> 3.5

**Pull request merged**